slow_tests/test_rbfe.py

```diff
@@ -53,7 +51,6 @@ def test_predict(self):
 
         ddg, results = model.predict(ordered_params, mol_a, mol_b, core, seed=2022)
         self.assertEqual(len(results), 2)
-        np.testing.assert_equal(ddg, 12.32189679129123)
```

Comment:
This value is significantly different on CI, about -0.5. Not sure about leaving it out, but wanted to avoid more if statements regarding os for these sorts of checks

Reply:
Interesting that this is an issue for test_rbfe but not test_rabfe... Taking a look.

Do we initially have a suggestion whether this is due to a random state issue specific to rbfe / this test, or something more generic different about CI environment (package version differences, OS differences)?

Reply:
Double-checking: equil_cache is freshly generated using the same random seed in both in local testing and CI?

https://github.com/proteneer/timemachine/blob/01b9cf14fc8f3d40777c3f6a99df3fd1d8721b8a/timemachine/fe/model.py#L152-L156

Reply:
For this test there is no equil cache being generated and the other test that runs stores the cache in a temporary directory to avoid impacting other tests. 

My initial thought is that the difference comes from OS differences, but I had thought that was only for the AM1 charges, which this shouldn't be impacted by using the simple charge forcefield. Will see if I can't determine whats happening here, as I would prefer this to have deterministic value/assertion

Reply:
>For this test there is no equil cache being generated

Ahh, right! I had misremembered the default is  https://github.com/proteneer/timemachine/blob/01b9cf14fc8f3d40777c3f6a99df3fd1d8721b8a/timemachine/fe/model.py#L35

>My initial thought is that the difference comes from OS differences, but I had thought that was only for the AM1 charges, which this shouldn't be impacted by using the simple charge forcefield. 

Agreed.

>Will see if I can't determine whats happening here, as I would prefer this to have deterministic value/assertion

Perhaps a fallback: `assert ddg in allowed_values`, `allowed_values = [12.32189679129123, 11.78817841291265]`, with a note about the unexplained local / CI difference.

(So it still passes locally and in CI, but would still fail if `ddg(seed, os_details)` is not a deterministic function or if relevant `os_details` change...)

Reply:
I decided to remove the changes specific to RBFE as it isn't immediately important to seed and I would like to dig into the source of this difference.

> Perhaps a fallback: assert ddg in allowed_values, allowed_values = [12.32189679129123, 11.78817841291265], with a note about the unexplained local / CI difference.

I do like this idea, and can put the changes back if you think it is important to change RABFE and RBFE models at the same time.

Reply:
>I do like this idea, and can put the changes back if you think it is important to change RABFE and RBFE models at the same time.

I don't see a reason to change the RABFE and RBFE models at the same time.